packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -58,7 +58,17 @@ export class WorkspaceFrontendContribution implements CommandContribution, Keybi
         const workspaceFolderCountKey = this.contextKeyService.createKey<number>('workspaceFolderCount', 0);
         const updateWorkspaceFolderCountKey = () => workspaceFolderCountKey.set(this.workspaceService.tryGetRoots().length);
         updateWorkspaceFolderCountKey();
-        this.workspaceService.onWorkspaceChanged(updateWorkspaceFolderCountKey);
+        this.workspaceService.onWorkspaceChanged(() => {
```

Comment:
`this.udpateStyles` should be called once before the listener as well to avoid missing an event

Reply:
oh, yes. Thanks for the hint...